SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SchemaGenerator.cpp

```diff
@@ -23,21 +23,23 @@ namespace
 {
 ESchemaComponentType PropertyGroupToSchemaComponentType(EReplicatedPropertyGroup Group)
 {
-	if (Group == REP_MultiClient)
+	switch (Group)
 	{
+	case REP_MultiClient:
 		return SCHEMA_Data;
-	}
-	else if (Group == REP_SingleClient)
-	{
+	case REP_SingleClient:
 		return SCHEMA_OwnerOnly;
-	}
-	else if (Group == REP_InitialOnly)
-	{
+	case REP_InitialOnly:
 		return SCHEMA_InitialOnly;
-	}
-	else
-	{
+	case REP_ServerOnly:
+		return SCHEMA_ServerOnly;
+	default:
 		checkNoEntry();
+		static_assert(
```

Comment:
Double static_assert here

Reply:
Oh they're asserting on different values

Reply:
This is intentional, one for REPs and one for SCHEMAs, but in general I may have gone a bit overboard with the static_asserts.